templates/web/base/common_header_tags.html

```diff
@@ -6,6 +6,10 @@
     id="js-meta-data" data-page="[% page %]"
     data-cobrand="[% c.cobrand.moniker %]">
 
+[% IF csrf_token %]
+  <meta content="[% csrf_token %]" name="csrf-token" />
```

Comment:
Switch the other things that could use this to it? e.g. offline.js has it once and staff.js line 109ish. And indeed the 400 handler in this PR :)